mcproto/decode.go

```diff
@@ -58,9 +60,65 @@ func DecodeLoginStart(data interface{}) (*LoginStart, error) {
 		return loginStart, errors.Wrap(err, "failed to read username")
 	}
 
-	loginStart.PlayerUuid, err = ReadUuid(buffer)
-	if err != nil {
-		return loginStart, errors.Wrap(err, "failed to read player uuid")
+	// These versions can send player keypair data. Ignore it.
+	// Reference: https://github.com/MCCTeam/Minecraft-Console-Client/blob/f785f509f228bf787c237ac139e6f666a960819a/MinecraftClient/Protocol/Handlers/Protocol18.cs#L2808-L2828
```

Comment:
Please also reference the mineraft wiki at protocol version page for 1.19

https://minecraft.wiki/w/Minecraft_Wiki:Projects/wiki.vg_merge/Protocol?oldid=2772902#Login_Start